PDK_Abstraction/FinFET14nm_Mock_PDK/FinFET_Mock_PDK_Abstraction_v1.json

```diff
@@ -0,0 +1,436 @@
+{
+"_comment":"The rules defined in this file are discussed in FinFET_MOCK_PDK.ppt",
+
+"Abstraction": [
+
+{
+"Layer": "Poly",
+"LayerNo": 1,
+"Direction": "V",
+"Width": [14],
+"Space": [66],
+"MinL": [60],
+"MaxL": ["NA"],
+"End-to-End": ["NA"],
+"Offset": [0]
+},
+
+{
+"Layer": "Fin",
+"LayerNo": 3,
+"Direction": "H",
+"Width": [10],
+"Space": [32],
+"MinL": [100],
+"MaxL": ["NA"],
+"End-to-End": ["NA"],
+"Offset": [0]
+},
+
+{
+"Layer": "M1",
+"LayerNo": 6,
+"Direction": "V",
+"Color": 2,
+"Width": [32],
+"Space": [48],
+"MinL": [180],
+"MaxL": ["NA"],
+"End-to-End": [48],
+"Offset": [40]
+},
+
+{
+"Layer": "V1",
+"LayerNo": 8,
+"Stack": "M1-M2",
+"WidthX": [32],
+"WidthY": [32],
+"SpaceX": [76],
+"SpaceY": [76],
+"VencA_L": [20],
+"VencA_H": [20],
+"VencP_L": [0],
+"VencP_H": [0],
+"MinNo": [1]
+},
+
+{
+"Layer": "M2",
+"LayerNo": 11,
+"Direction": "H",
+"Color": 2,
+"Width": [32],
+"Space": [52],
+"MinL": [200],
+"MaxL": ["NA"],
+"End-to-End": [48],
+"Offset": [42]
+},
+
+{
+"Layer": "V2",
+"LayerNo": 13,
+"Stack": "M2-M3",
+"WidthX": [32],
+"WidthY": [32],
+"SpaceX": [76],
+"SpaceY": [76],
+"VencA_L": [20],
+"VencA_H": [20],
+"VencP_L": [0],
+"VencP_H": [4],
+"MinNo": [1]
+},
+
+{
+"Layer": "M3",
+"LayerNo": 15,
+"Direction": "V",
+"Color": 2,
+"Width": [40],
+"Space": [40],
+"MinL": [210],
+"MaxL": ["NA"],
+"End-to-End": [48],
+"Offset": [40]
+},
+
+{
+"Layer": "V3",
+"LayerNo": 20,
+"Stack": "M3-M4",
+"WidthX": [40],
+"WidthY": [40],
+"SpaceX": [76],
+"SpaceY": [76],
+"VencA_L": [20],
+"VencA_H": [20],
+"VencP_L": [0],
+"VencP_H": [0],
+"MinNo": [1]
+},
+
+{
+"Layer": "M4",
+"LayerNo": 23,
+"Direction": "H",
+"Color": 1,
+"Width": [40],
+"Space": [40],
+"MinL": [140],
+"MaxL": ["NA"],
+"End-to-End": [65],
+"Offset": [0]
+},
+
+{
+"Layer": "V4",
+"LayerNo": 25,
+"Stack": "M4-M5",
+"WidthX": [40],
+"WidthY": [40],
+"SpaceX": [76],
+"SpaceY": [76],
+"VencA_L": [20],
+"VencA_H": [20],
+"VencP_L": [0],
+"VencP_H": [12],
+"MinNo": [1]
+},
+
+{
+"Layer": "M5",
+"LayerNo": 28,
+"Direction": "V",
+"Color": 1,
+"Width": [64],
+"Space": [80],
```

Comment:
Also, I see Pitch has been replaced by Space. Not sure what is the more common convention. End-To-End is also a type of space.

Reply:
In my view, "Space" gives more flexibility in abstracting the rules. Yes, End-To-End is also a type of space between the edges of layers.